Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -468,6 +464,29 @@ private bool ShouldStartBrowser()
             return true;
         }
 
+        private string GetBrowserPath()
+        {
+            ThreadHelper.ThrowIfNotOnUIThread();
+
+            var uiShellOpenDocument = (IVsUIShellOpenDocument)ServiceProvider.GlobalProvider.GetService(typeof(SVsUIShellOpenDocument));
+            uiShellOpenDocument.GetFirstDefaultPreviewer(out var defaultBrowserPath, out _, out _);
+
+            return defaultBrowserPath;
+        }
+
+        private static void OpenBrowser(string browserPath, string webBrowserUrl)
+        {
+            // Chrome has known issues with being launched as admin.
+            if (!string.IsNullOrEmpty(browserPath) && browserPath.EndsWith(NodejsConstants.ChromeApplicationName))
+            {
+                SystemUtility.ExecuteProcessUnElevated(browserPath, webBrowserUrl);
+            }
+            else
+            {
+                VsShellUtilities.OpenBrowser(webBrowserUrl, (uint)__VSOSPFLAGS.OSP_LaunchNewBrowser);
```

Comment:
I assume we already don't support attaching for client-side script debugging on launch, and thus this has no impact on debugging client side script (though may complicate matters if we want to support that later).